ambry-network/src/main/java/com.github.ambry.network/Selector.java

```diff
@@ -548,3 +562,16 @@ private SocketChannel channel(SelectionKey key) {
     return (SocketChannel) key.channel();
   }
 }
+
+/**
+ * Class used to store {@link SSLTransmission} which are yet to complete its handshake, along with start time of the same
+ */
+class PendingHandshakeTransmission {
```

Comment:
would prefer naming this `TransmissionPendingHandshake` and this should be a private class.